packages/sdks/web-component/src/lib/types.ts

```diff
@@ -141,6 +141,11 @@ export type CustomScreenState = Omit<
     type: ScreenState['errorType'];
   };
   action?: string;
+  inboundAppApproveScopes?: {
```

Comment:
I'm a bit concerned about this type, why do we need this? maybe we can have it something more generic? like `& Record<string, any>`?

Reply:
I'm not sure I understand
This is the type we currently have
Why should we make it more generic?

Reply:
it seems a bit obsecure to me that `CustomScreenState` has a very specific property which is `inboundAppApproveScopes`

Reply:
We agreed that the custom screen state will be an object with a flat structure, so I'm following this structure
Regarding the name, this is what they need to send back when calling next, so we decided to use the same name

What do you have in mind?